pkg/controller/catalog/util/converter.go

```diff
@@ -17,40 +17,53 @@ limitations under the License.
 package util
 
 import (
-	"encoding/json"
-
 	sbmodel "github.com/kubernetes-incubator/service-catalog/model/service_broker"
-	scmodel "github.com/kubernetes-incubator/service-catalog/model/service_controller"
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
 )
 
 // ConvertCredential converts a Credential object from the broker model to the
 // controller model.
-func ConvertCredential(in *sbmodel.Credential) (*scmodel.Credential, error) {
-	j, err := json.Marshal(in)
-	if err != nil {
-		return nil, err
-	}
+// func ConvertCredential(in *sbmodel.Credential) (*servicecatalog.BindingSpec, error) {
```

Comment:
I believe so. This is vestigial from the hand-off from @arschles to me. I will delete it and amend the PR.